yarn-project/end-to-end/src/e2e_cheat_codes.test.ts

```diff
@@ -49,5 +56,67 @@ describe('e2e_cheat_codes', () => {
       expect(await cc.l1.blockNumber()).toBe(blockNumber + 1);
       expect(await cc.l1.timestamp()).toBe(timestamp + increment);
     });
+
+    it('getStorageAt', async () => {
+      const res = await cc.l1.getStorageAt(EthAddress.fromString('0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266'), '0x0');
+      expect(res).toBe('0x0000000000000000000000000000000000000000000000000000000000000000');
+    });
+
+    it.each(['1', 'bc40fbf4394cd00f78fae9763b0c2c71b21ea442c42fdadc5b720537240ebac1'])(
+      'setStorageAt',
+      async storageSlot => {
+        const storageSlotInHex: `0x${string}` = `0x${storageSlot}`;
+        const valueToSet = '0x0000000000000000000000000000000000000000000000000000000000000005';
+        const contractAddress = EthAddress.fromString('0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266');
+        await cc.l1.setStorageAt(contractAddress, storageSlotInHex, valueToSet);
```

Comment:
Can you add a test where the storage slot is not provided in hex, but as a `bigint` instead or even a buffer. We might want to add a `keccak` function to the cheatcodes for easy access to compute storage slots.